lib/src/callable.dart

```diff
@@ -117,4 +120,14 @@ abstract class Callable extends AsyncCallable {
   factory Callable.function(String name, String arguments,
           Value callback(List<Value> arguments)) =>
       BuiltInCallable.function(name, arguments, callback);
+
+  /// Creates a host callable with a single [signature] and a single [callback].
```

Comment:
"host callable" isn't really a meaningful term in the context of the Dart Sass API. I think it would make more sense to name/document this after its function, like `fromSignature()`.